scripts/clang_format_all.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash
```

Comment:
should we not have this file still around and make it check for `git-clang-format` and run it on the root?

Reply:
Good idea. I added a cleaned up version back in 👍